openbb_terminal/econometrics/econometrics_controller.py

```diff
@@ -471,15 +471,15 @@ def call_export(self, other_args: List[str]):
         if other_args and "-" not in other_args[0][0]:
             other_args.insert(0, "-n")
         ns_parser = self.parse_known_args_and_warn(
-            parser, other_args, export_allowed=NO_EXPORT
+            parser, other_args, export_allowed=EXPORT_BOTH_RAW_DATA_AND_FIGURES
```

Comment:
The parser is defined, in the lines above, with constrained `choices`.  Without including your changes, I can comment out line 466 and I can set the filename accordingly.

Reply:
I have added the `--export` option, which I observed being used in the following format:
`/stocks/ $ load aapl --start 2022-01-01 --end 2022-12-31 --weekly --export aapl.xlsx --sheet-name weekly`

Now, the export command in the `econometrics` section will resemble the following:
`/econometrics/ $ export aapl --export test.xlsx`

Reply:
Found a better solution.
Added a -f/--file option to provide the filename for export.
Made -t/--type and -f/--file mutually exclusive.
If -f is not given, -t is required and the filename will be generated based on the time and module.